source/simulator/helper_functions.cc

```diff
@@ -315,6 +315,20 @@ namespace aspect
 
 
 
+  template <int dim>
+  void
+  Simulator<dim>::
+  compute_pressure_scaling_factor()
+  {
+    // determine how to treat the pressure. we have to scale it for the solver
+    // to make velocities and pressures of roughly the same (numerical) size,
+    // and we may have to fix up the right hand side vector before solving for
+    // compressible models if there are no in-/outflow boundaries
```

Comment:
How does the second part of the second relate to this function?

Reply:
Not sure, actually. I copied it from the original statement, but I've taken that half sentence out now.